sdk/core/core-http/package.json

```diff
@@ -118,10 +118,9 @@
     "@azure/core-auth": "^1.3.0",
     "@azure/core-tracing": "1.0.0-preview.13",
     "@azure/logger": "^1.0.0",
-    "@types/node-fetch": "^2.5.0",
```

Comment:
Removing `@types/node-fetch` could potentially break customers who depending on the implicitly introduced transitive dependency on `@types/node`